test/catch2_test_printing.cu

```diff
@@ -0,0 +1,34 @@
+#include "test_util.h"
+
+#include "catch2_test_helper.h"
+
+template <typename T>
+std::string print(T val) 
+{
+  std::stringstream ss;
+  ss << val;
+  return ss.str();
+}
+
+#if CUB_IS_INT128_ENABLED
+TEST_CASE("Test utils can print __int128", "[test][utils]")
+{
+  REQUIRE( print(__int128_t{0}) == "0" );
+  REQUIRE( print(__int128_t{-1}) == "-1" );
```

Comment:
Could you also print some strictly positive numbers, and some numbers that don't fit into 64 bits?